### DISCUSSION TITLE
Runtime Error in Code

### DISCUSSION THREAD

**Discussion by labyrinth2** [jax-ml/jax#13725] [Q&A]
```python

```

my code above is having a run time error and i don't know what will i do to modify it. my professor said that "t" can someone help me to modify my code according to what my prof. said?

**Comment by jakevdp** [comment]
Hi - it looks like this is a general Python programming question. This is not the appropriate forum to ask such questions; I'd suggest asking on http://stackoverflow.com or a similar site. If you have questions specific to the JAX project, please feel free to raise them here!
